featurevis/image_reader.py

```diff
@@ -15,7 +16,7 @@ def save_npy_as_png(path):
     """
     filepath = Path(path)
     filename = filepath.stem
-    out_path = Path("{0}.png".format(filename))
+    out_path = Path("outputs/{0}.png".format(filename))
```

Comment:
I think this should not be changed this way. With the current implementation of taking a path and putting the file there, the user can precisely decide where he wants the output to go. If we prefix with `output/`, then it always needs to be put in a folder called output, which I think is not necessary.

Reply:
Probably it would be better for future users. I added this for having an order for myself since I always save the images in my repo folder. I will make the output path freely choosable.